internal/core/graphics/image.rs

```diff
@@ -399,6 +430,8 @@ pub enum ImageInner {
     #[cfg(not(target_arch = "wasm32"))]
     BorrowedOpenGLTexture(BorrowedOpenGLTexture) = 6,
     NineSlice(vtable::VRc<OpaqueImageVTable, NineSliceImage>) = 7,
+    #[cfg(feature = "unstable-wgpu-24")]
+    WGPUTexture(vtable::VRc<OpaqueImageVTable, WGPUTexture>) = 8,
```

Comment:
Alternatives:

```suggestion
    WGPU24Texture(vtable::VRc<OpaqueImageVTable, WGPU24Texture>) = 8,
```

Or just

```suggestion
    WGPU24Texture(vtable::VRc<OpaqueImageVTable, wgpu_24::Texture>) = 8,
```

I'm not sure how this will play out when we support several version of wgpu in parallel.
The tradeoff is having a bit more condition over the image module, or more complicated conditions.

Reply:
My preference is to keep impact of multiple versions as minimal as possible, hence the `WGPUTexture` indirection here, so that adding a variant doesn't require touching all of this code in core again.